Fastfile

```diff
@@ -100,6 +100,10 @@ def _build_number
   end
 end
 
+def _get_project_build_number
+  return ENV.include?('TAB_TARGET_FOR_BUILD_NUMBER') ? get_version_number(target: ENV['TAB_TARGET_FOR_BUILD_NUMBER']) : get_version_number
```

Comment:
I wonder if instead of `TAB_TARGET_FOR_BUILD_NUMBER` we have something more generic like `TAB_PRIMARY_TARGET`?

Reply:
As you wish really, although I don't see the need.
> No need to try solving a problem which doesn't exist